README.md

```diff
@@ -2,145 +2,27 @@
-The test users listed in [the develop section](https://www.kimai.org/documentation/installation.html) also exist.
-
-```bash
-docker run --rm -ti -p 8001:8001 --name kimai2 kimai/kimai2:apache-debian-1.8-dev
-docker exec kimai2 rm /opt/kimai/var/data/kimai.sqlite
-docker exec kimai2 /opt/kimai/bin/console kimai:reset-dev
-docker exec kimai2 /opt/kimai/bin/console kimai:create-user admin admin@example.com ROLE_SUPER_ADMIN changemeplease
-```
-
-### Production
-
-Run a production kimai with persistent database in a seperate mysql container.
-The best way of doing this is with a docker compose file.
-You can hit kimai at `http://localhost:8001` and log in with `superadmin` / `changemeplease`.
-
-```yaml
-version: '3.5'
-services:
-
-  sqldb:
-    image: mysql:5.7
-    environment:
-      - MYSQL_DATABASE=kimai
-      - MYSQL_USER=kimaiuser
-      - MYSQL_PASSWORD=kimaipassword
-      - MYSQL_ROOT_PASSWORD=changemeplease
-    volumes:
-      - /var/lib/mysql
-    command: --default-storage-engine innodb
-    restart: unless-stopped
-    healthcheck:
-      test: mysqladmin -pchangemeplease ping -h localhost
-      interval: 20s
-      start_period: 10s
-      timeout: 10s
-      retries: 3
-
-  nginx:
-    build: compose
-    ports:
-      - 8001:80
-    volumes:
-      - ./nginx_site.conf:/etc/nginx/conf.d/default.conf:ro
-      - public:/opt/kimai/public:ro
-    restart: unless-stopped
-    depends_on:
-      - kimai
-    healthcheck:
-      test:  wget --spider http://nginx/health || exit 1
-      interval: 20s
-      start_period: 10s
-      timeout: 10s
-      retries: 3
-
-  kimai:
-    image: kimai/kimai2:fpm-alpine-1.8-prod
-    environment:
-      - APP_ENV=prod
-      - TRUSTED_HOSTS=localhost
-      - ADMINMAIL=admin@kimai.local
-      - ADMINPASS=changemeplease
-    volumes:
-      - public:/opt/kimai/public
-      - var:/opt/kimai/var
-      # - ./ldap.conf:/etc/openldap/ldap.conf:z
-      # - ./ROOT-CA.pem:/etc/ssl/certs/ROOT-CA.pem:z
-    restart: unless-stopped
-    healthcheck:
-      test: wget --spider http://nginx || exit 1
-      interval: 20s
-      start_period: 10s
-      timeout: 10s
-      retries: 3
-
-  postfix:
-    image: catatnight/postfix:latest
-    environment:
-      maildomain: kimai.local
-      smtp_user: kimai:kimai
-    restart: unless-stopped
-
-volumes:
-    var:
-    public:
-```
-
-## Permissions
-
-If you are mounting the code base into the container (`-v $PWD/kimai:/opt/kimai`) then you will need to fix the permissions on the var folder.
-
-```bash
-docker exec --user root CONTAINER_NAME chown -R www-data:www-data /opt/kimai/var
-```
-
-or
-
-```bash
-docker-compose --user root exec CONTAINER_NAME chown -R www-data:www-data /opt/kimai/var
-```
-
-## Runtime Arguments
-
-The following settings can set at runtime:
-
-Kimai/symfony core settings, see the symfony and kimai docs for more info on these.
+Run the latest master version of Kimai in production mode using a bundled sqlite DB:
```

Comment:
Even though this might be the simplest use-case, I would argue that we should NOT promote the use of SQLite, as it proved to lead to all kind of bugs (due to the missing support of cascade deletes / foreign keys).

You know: users read the docs only as much as needed and then immediately stop when they see a working Kimai^^

Reply:
Thanks for adding the hint about "no production usage"!